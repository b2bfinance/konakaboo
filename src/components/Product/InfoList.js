import styled from 'styled-components';

export default styled.ul`
  list-style: none;
  margin-bottom: 0;
  margin-top: 1rem;
  padding: 0;
`;
