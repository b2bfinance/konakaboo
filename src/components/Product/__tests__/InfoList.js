import React from 'react';
import { render } from 'enzyme';
import InfoList from '../InfoList';

describe('InfoList', () => {
  test('renders correctly', () => {
    expect(render(<InfoList />)).toMatchSnapshot();
  });
});
