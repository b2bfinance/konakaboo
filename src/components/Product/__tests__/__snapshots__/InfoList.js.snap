// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoList renders correctly 1`] = `
.c0 {
  list-style: none;
  margin-bottom: 0;
  margin-top: 1rem;
  padding: 0;
}

<ul
  class="c0"
>
  
</ul>
`;
